### PR TITLE
Front-End Architecture & HTML

### PR DESCRIPTION
**Pull request by Hyunsujk**
worksheet for Front-End Architecture & HTML

### CODE REVIEW

**Comment by Hyunsujk** [comment]
I have 2 mr open for answer key update 
https://github.com/KansasCityWomeninTechnology/AnswerKeys/pull/2
https://github.com/KansasCityWomeninTechnology/AnswerKeys/pull/3

File: docs/html/1-intro/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+> [!WARNING]
+> Before starting the worksheet, please take a moment to review the [Setup instructions](../setup/?id=setup) to ensure you have all the tools and workspace setup you need for today's work.
+
+> [!WARNING]
+> If you are using StackBlitz, please skip to [StackBlitz instructions](./?id=cloud-ide) below.
+
+# Create the HTML file
+
+<!-- [filename](./1file.md ':include') -->
+<!-- Contents from file. make changes in external files and copy here until bug is fixed -->
```
Comment (alisaduncan) [inline docs/html/1-intro/README.md:18]: This bug was fixed by docsify, so you can load markdown from external files now. Please feel to utilize this feature as you'd like. :)

File: docs/html/setup/README.md [hunk 2]
```diff
@@ -1,44 +1,43 @@
...
-
+> Navigate to GitHub to create a personal account or log in. Feel free to ask your mentor for help! When complete and logged in to GitHub, return here to continue the instructions. You won't install any tools listed on this page, but please read through them so you have context for today's worksheet.
 
 # Required tools for this session
 
-You don't need all the tools for tonight's session. If you haven't installed all the tools listed in [The Tools](https://bit.ly/CnCTheTools) guide already, use the following list to install only the required tools for the worksheet:
-
->[!WARNING]If you are borrowing a laptop, your tools are pre-installed. Please ask your mentor to explain the tools we are using tonight.
-
+You don't need all the tools for today's session. If you haven't installed all the tools listed in [The Tools](https://bit.ly/CnCTheTools) guide already, use the following list to install only the required tools for the worksheet:
```
Comment (alisaduncan) [inline docs/html/setup/README.md:19]: You might want to consider removing this so tools can be approached session by session. And then you'll need to bring in any tools directly into here. Up to you on how you want to handle, but if there's the big tools section, the expectation is that defines all the tools needed for the entire set of sessions.

File: docs/html/setup/README.md [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
 
-1. [Chrome browser](/tools/browser/ ':target=_blank')
+> [!WARNING]If you are borrowing a laptop, your tools are pre-installed. Please ask your mentor to explain the tools we are using today.
```
Comment (alisaduncan) [inline docs/html/setup/README.md:21]: Just to make sure, have you verified these all render correctly? some of these plugins have a specific white-space requirement

File: docs/tools/vscode/README.md [hunk 4]
```diff
@@ -33,19 +32,19 @@ If you have ever downloaded and installed a custom font to use in your word proc
 
 We will install required extensions and list optional extensions you can install at the end of the page to add a little pizzazz to your coding experience.
 
-# Required Extensions
+# Required Extensions :id=required-extensions
 
 Install the required extensions:
 
-* [Angular Essentials](https://marketplace.visualstudio.com/items?itemName=johnpapa.angular-essentials) - An all in one extension covering the needs for Angular developers
+- [Angular Essentials](https://marketplace.visualstudio.com/items?itemName=johnpapa.angular-essentials) - An all in one extension covering the needs for Angular developers
```
Comment (alisaduncan) [inline docs/tools/vscode/README.md:39]: I don't think we're planning on using Angular, if we even finish out this curriculum session. See above notes about tools

**Review by alisaduncan — COMMENTED** [review]
A few comments. Are you planning on adding a follow up PR with the updates to the main page and navigation?

File: docs/html/1-intro/README.md (hunk 1 above)
Comment (Hyunsujk) [reply docs/html/1-intro/README.md:18]: gotcha. I left those comments from the previous worksheet. I don't think I need those so probably delete them.

File: docs/html/setup/README.md (hunk 3 above)
Comment (Hyunsujk) [reply docs/html/setup/README.md:21]: It was when I checked and seems like now it doesn't and get formatted when save🙃

**Pull request merged**